templates/spring-kotlin/files/.platform.app.yaml

```diff
@@ -26,4 +26,15 @@ relationships:
 # The configuration of app when it is exposed to the web.
 web:
     commands:
-        start:  java -jar -Xmx$(jq .info.limits.memory /run/config.json)m -XX:+ExitOnOutOfMemoryError target/spring-kotlin.jar --server.port=$PORT
+        start:  |
+           export DB_PORT=`echo $PLATFORM_RELATIONSHIPS|base64 -d|jq -r ".database[0].port"`
```

Comment:
We have these in other templates in a `.environment` file - you want to move the export block there for consistency?

Reply:
Yeap, I liked the idea.